Modix/ClientApp/src/views/UserLookup.vue

```diff
@@ -1,12 +1,13 @@
 <template>
-    <div>
+    <section class="section">
+        <h1 class="title">User Lookup</h1>
```

Comment:
Is this supposed to be left-aligned? Looks kinda weird.

![image](https://user-images.githubusercontent.com/2829282/55645726-49c03900-57a7-11e9-825c-0d51e16d63f6.png)

Reply:
Oops, that's what I get for developing with only 1/3 of my monitor. Will fix.